src/functionalTest/java/uk/gov/hmcts/reform/sendletter/controllers/SendLetterTest.java

```diff
@@ -60,7 +60,7 @@
     @Mock
     private IQueueClient queueClient;
 
-    @MockBean
+    @SpyBean
```

Comment:
Should we remove `MockTransactionManager` as I believe we would have real transaction?

Reply:
win. :shipit:

Reply:
Also I had one query we are still doing nothing when `repository.save(..) `is called.If we are using test containers we would want to record to be inserted in DB ? 
```
doNothing()
            .when(letterRepository)
            .save(any(DbLetter.class), any(Instant.class), anyString());
```